dummy_integrator/plugin.cpp

```diff
@@ -0,0 +1,63 @@
+#include "common/plugin.hpp"
+#include "common/phonebook.hpp"
+#include "common/switchboard.hpp"
+#include "common/data_format.hpp"
+
+using namespace ILLIXR;
+
+class dummy_integrator : public plugin {
```

Comment:
Hmm, that's odd because `Eigen::Matrix<double,3,1> w_hat;` should result in uninitialized coefficients according to the Eigen spec. In any case, this is not safe. We should explicitly initialize the IMU readings: all zeros for angular velocity and (0, 0, 9.81) for acceleration.

Reply:
You are correct. The code has undefined behavior. This means it _may not_ work not that it _must not_. This is probably why it empirically works.

I'll explicitly zero this out in the next iteration.

Reply:
Should we explicitly set the 9.81 for acceleration?